fh-messaging/lib/services/mbaas_metrics.js

```diff
@@ -215,7 +215,10 @@ module.exports = function(mongodb) {
       logger.info("Rollup options = " + JSON.stringify(options));
       collection.mapReduce(mReduceConf.map, mReduceConf.reduce, options, function(err,results) {
         var filteredErr = helpers.ignoreSomeErrs(err);
-        return callback(filteredErr, results);
+        collection.find({}, function(err, docs) {
+          console.log('@@@@@doooocs', docs);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AS2228)